include/odp/api/spec/classification.h

```diff
@@ -176,6 +176,27 @@ typedef struct odp_bp_param_t {
 	odp_threshold_t threshold;
 } odp_bp_param_t;
 
+/**
+ * Classifier CoS specific statistics counters
+ *
+ * Counters are incremented per packet classified to the CoS.
```

Comment:
This sentence needs clarification: if all (stats enabled) CoS'es on the way from default CoS to the "resulting" CoS increment their counters for a packet, or if only the "resulting" CoS increments its counters.

Reply:
Chose the former alternative in v9.